docs/dev/analysis/features/table/table-cell.rst

```diff
@@ -29,6 +43,15 @@ MS API - Partial Summary
 * WordWrap
 
 
+WD_ALIGN_VERTICAL Enumeration
+---------------------------------
+
+* wdAlignVerticalTop (0) Top vertical alignment.
+* wdAlignVerticalCenter (1) Center vertical alignment.
+* wdAlignVerticalJustify (2) Justified vertical alignment.
```

Comment:
@wich I'm not finding this Justify option in the MS-defined enumeration, although I do see that 'both' is a defined option for the `w:val` attribute value. Did you find the wdAlignVerticalJustify member defined somewhere or is this just a best guess so far of what that indicates as far as behavior in Word?

Also, have you tried out what a table cell having the 'both' value does? I'm supposing it just ignores that particular value.